torch/csrc/distributed/c10d/intra_node_comm.cpp

```diff
@@ -278,7 +277,7 @@ std::vector<T> storeAllGather(
     store->wait({peerKeys[r]});
     auto payload = store->get(peerKeys[r]);
     TORCH_CHECK(payload.size() == sizeof(T));
-    T peerVal;
+    T peerVal{};
```

Comment:
Is T guaranteed to be default constructible? If not, seems like this could be a false positive.

Reply:
No, T is replaced as POD structures in the file and there is no type trait on it.